src/ui/SearchAlerts/FollowItem.ts

```diff
@@ -96,8 +111,10 @@ export class FollowItem extends Component {
   }
```

Comment:
No, it's not really relevant for an end user perspective.

I'm not even sure why it's public in the first place...